packages/data-stores/src/reader/constants.ts

```diff
@@ -21,3 +21,8 @@ export enum SiteSubscriptionsSortBy {
 	LastUpdated = 'last_updated',
 	DateSubscribed = 'date_subscribed',
 }
+
```

Comment:
I'm adding the same comment as with the Filtering patch: I don't know if this should be in the reader package. We used the reader package for subscription management because it had to live in two places. But I don't see Subscribers being that related to Reader to be honest.

Reply:
That's true. I'll change it 👍